doc/args.rst

```diff
@@ -34,5 +34,9 @@ add comments on found issues right to the selected code review system.
         | * -f='test 1:!unit test 1'    - run all steps with 'test 1' substring in their names except those
          containing 'unit test 1'
 
+    --html-log -hl : @after
+        To integrate this log into Jenkins job with ``htmlpublisher`` plugin,
```

Comment:
it's not clear at all from here why would I want to do that

Reply:
How about
"Generates user-friendly self-contained HTML log. To integrate this log with blabla see blabla"
?